test/test.js

```diff
@@ -15,7 +15,7 @@ function doesNotInclude(string, substring) {
 }
 
 function makeBundle(entries) {
-  return rollup({ entry: entries, plugins: [multiEntry()] });
+  return rollup({ input: entries, plugins: [multiEntry()] });
```

Comment:
I updated the tests as best I could, but was unable to run them due to some third-party dependency it seems on Facebooks Flow.